### DISCUSSION TITLE
No logs and no pings

### DISCUSSION THREAD

**Discussion by Coldfirex** [Security-Onion-Solutions/securityonion#11516] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Does not meet minimum requirements

### CPU

6

### RAM

15

### Storage for /

146899900

### Storage for /nsm

324696564

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Howdy,
First time setting up SO.  I have web access to SO and have setup a dedicated span port to a second nic which is piped through Proxmox to my SO VM.  
My issues are that I cannot ping SO's Ip address even though I can pull up its web gui, and I have zero logs shown in SO and Kibana.  Not sure where to start as I have minimal linux experience.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Acewiza** [comment]
When you say you "cannot ping" exactly what ping response _do_ you get?  There's lotsa different ways things don't work.

**Reply by Coldfirex** [reply]
'Destination host unreachable'
SO itself can ping anywhere.  Its odd, like the firewall is blocking it, but I dont know where to try allowing icmp inbound.

**Comment by Acewiza** [comment]
Well, that's certainly un-expected.  Maybe the guys more familiar with the VM stuff will chime in...

**Comment by dougburks** [comment]
15GB RAM doesn't meet the minimum requirements for a Standalone installation. From https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments:
> In a standalone deployment, the manager components and the sensor components all run on a single box, therefore, your hardware requirements will reflect that. You’ll need at minimum 24GB RAM, 4 CPU cores, and 200GB storage. At the bare minimum of 24GB RAM, you would most likely need swap space to avoid issues.

**Reply by Coldfirex** [reply, marked as answer]
Howdy Doug,
I actually was able to resolve this earlier today.  I have more ram coming, just waiting on it to be delivered.
I attempted reinstalling a few times from latest ISO without any change.  I noticed that the installer mentioned it would wipe the disk.  I have a feeling it doesnt actually wipe the whole disk though.  After 3 reinstalls, I deleted the virtual disk, created a new one, and after another installation all was well.

**Comment by m0duspwnens** [comment]
> My issues are that I cannot ping SO's Ip address even though I can pull up its web gui

SO nodes will respond to ping in 2.4.30.
